client/src/pages/Login.js

```diff
@@ -1,8 +1,11 @@
 import React from "react";
+import Login from "../Components/Login";
```

Comment:
Since you are defining a function here called Login to export this page, you cannot also name the import of the component login as well. 
This name has to change or it won't render.